recipes/connection_pool/meta.yaml

```
@@ -0,0 +1,40 @@
+{% set name = "ConnectionPool" %}
+{% set author = "zhouyl" %}
+{% set version = "0.0.3" %}
+{% set sha256 = "3acb9bbd6ef9a6ca9d5e056c0afd14dbc5ca8d610d788b162412f6235a6e3759" %}
+
+package:
+  name: connection_pool
+  version: {{ version }}
+
+source:
+    fn: {{ name }}.{{ version }}.tar.gz
+    url: https://github.com/{{ author }}/{{ name }}/archive/{{ version }}.tar.gz
+    sha256: {{ sha256 }}
```

Can we get this from PyPI instead?
```suggestion
    url: https://pypi.io/packages/source/c/{{ name }}/{{ name }}-{{ version }}.tar.gz
    sha256: bf429e7aef65921c69b4ed48f3d48d3eac1383b05d2df91884705842d974d0dc
```

---

Sure